src/main/java/bio/terra/workspace/service/workspace/flight/CreateGoogleContextRBSFlight.java

```diff
@@ -1,24 +1,44 @@
 package bio.terra.workspace.service.workspace.flight;
 
+import bio.terra.cloudres.google.billing.CloudBillingClientCow;
+import bio.terra.cloudres.google.cloudresourcemanager.CloudResourceManagerCow;
 import bio.terra.stairway.Flight;
 import bio.terra.stairway.FlightMap;
 import bio.terra.stairway.RetryRule;
 import bio.terra.stairway.RetryRuleExponentialBackoff;
+import bio.terra.workspace.db.WorkspaceDao;
 import bio.terra.workspace.service.buffer.BufferService;
 import org.springframework.context.ApplicationContext;
+import org.springframework.transaction.support.TransactionTemplate;
 
 // NOTE: DO NOT USE. Currently just a shell class to exercise connection to Buffer Service.
 public class CreateGoogleContextRBSFlight extends Flight {
   public CreateGoogleContextRBSFlight(FlightMap inputParameters, Object applicationContext) {
     super(inputParameters, applicationContext);
     ApplicationContext appContext = (ApplicationContext) applicationContext;
     BufferService bufferService = appContext.getBean(BufferService.class);
```

Comment:
How the bean get Buffer Service? When we create the BufferService, where is the code to set access token in request header?

Reply:
Example code without dependency injection: https://github.com/DataBiosphere/terra-resource-buffer/blob/mm-PF-237-use-test-runner-library/buffer-clienttests/src/main/java/scripts/utils/BufferServiceUtils.java#L55-L67

With Spring DI, that should be a bean like: https://github.com/DataBiosphere/terra-workspace-manager/blob/dev/src/main/java/bio/terra/workspace/app/configuration/spring/BeanConfig.java#L33